### DISCUSSION TITLE
Edit pixel icons in webUI

### DISCUSSION THREAD

**Discussion by rknotter** [pixelit-project/PixelIt#345] [Q&A]
Hi
I can't find a way to edit an existing icon in the webUI? If I paste the existing array in the text field nothing happens unfortunately. The draw field is not updated. Tried chrome, firefox and even edge... 
Am I missing something or is this not possible? 

Thanks all!

**Comment by Gardanjerte** [comment]
Hello,
I remember that it worked in the past, but i am not fully sure.
Now, it does not work.
Also interested in this feature. 
Regards!

**Comment by andrewjswan** [comment]
https://github.com/pixelit-project/PixelIt.WebUI/issues/64
